eras/alonzo/impl/cardano-ledger-alonzo.cabal

```diff
@@ -69,7 +69,6 @@ library
     cardano-crypto-class,
     cardano-ledger-core,
     cardano-ledger-shelley-ma,
-    cardano-prelude,
```

Comment:
Strange that this wasn't included in the earlier commit which removes the dependency on cardano-prelude. Interesting that it compiled with the dependency even though it was unnecessary (unless it became unnecessary because of something in this particular commit :thinking: )

Reply:
See my other comment about re-introducing our dependency on Cardano-prelude. In the end it only cherry-picked some parts. For the ledger/1.1.x branch I hope that's ok.